wbstack/sync.sh

```diff
@@ -46,8 +46,10 @@ $BASEDIR/sync/pacman
 $BASEDIR/sync/03-less-files.sh
 
 # TODO: Do this better / decide where these files should live and how they should be copied into 'dist'
-echo "Copy composer files into the 'dist' dir"
-cp -t ./dist/ composer.*.json composer.lock
+echo "Copy required files into the 'dist' dir"
+cp -t ./dist/ composer.*.json composer.lock health.php LocalSettings.php robots.txt
```

Comment:
We do not want to copy the composer.lock file from the root of the repo.
We just do not want to delete the file from the dist directory when re syncing